include/openssl/tls1.h

```diff
@@ -28,7 +28,11 @@ extern "C" {
 
 /* Default security level if not overridden at config time */
 # ifndef OPENSSL_TLS_SECURITY_LEVEL
-#  define OPENSSL_TLS_SECURITY_LEVEL 2
+#  if defined(OPENSSL_NO_TLS1_2) && defined(OPENSSL_NO_TLS1_3)
+#   define OPENSSL_TLS_SECURITY_LEVEL 0
+#  else
+#   define OPENSSL_TLS_SECURITY_LEVEL 2
+#  endif
```

Comment:
I am not sure this is right fix. You still have DTLS-1.2, don't you?

Reply:
good point...

Reply:
I've reverted that commit.
Except the possibility of working DTLS-1.2
This was also bypassed by "./config no-tls1_2 no-ec no-dh"
I'll add that configuration instead and see if it breaks different things...